MainAnalysis/20250310_HFUPCEEC/makefile

```diff
@@ -0,0 +1,13 @@
+default: ExecuteHFUPCEEC
+
+ExecuteHFUPCEEC: HFUPCEEC.cpp
+	g++ HFUPCEEC.cpp -O3 -I. -I$(ProjectBase)/CommonCode/include -I./include \
+		-o ExecuteHFUPCEEC `root-config --cflags --glibs` \
+		-I$(ProjectBase)/CommonCode/include \
+		$(ProjectBase)/CommonCode/library/*.o
```

Comment:
This inserts all the objects into the executable. If there are too many *.o's, the executable will be very large. If the size gets too big, we can enumerate them here to have only the ones needed

Reply:
Sure, for now I think it is a manageable size, but if it gets too big we can do that!